composer.json

```diff
@@ -41,13 +41,13 @@
     "install-codestandards": [
       "Dealerdirect\\Composer\\Plugin\\Installers\\PHPCodeSniffer\\Plugin::run"
     ],
-    "phpcs-src": "vendor/bin/phpcs",
-    "phpcs-tests": "vendor/bin/phpcs --runtime-set testVersion 5.6 tests/phpunit/",
+    "phpcs-src": "\"vendor/bin/phpcs\"",
+    "phpcs-tests": "\"vendor/bin/phpcs\" --runtime-set testVersion 5.6 tests/phpunit/",
     "phpcs": [
       "@phpcs-src",
       "@phpcs-tests"
     ],
-    "test-unit": "vendor/bin/phpunit --testsuite unit",
-    "test-integration": "vendor/bin/phpunit --testsuite integration"
+    "test-unit": "\"vendor/bin/phpunit\" --testsuite unit",
+    "test-integration": "\"vendor/bin/phpunit\" --testsuite integration"
```

Comment:
The backslashes are for Windows machines.  These changes work on both Mac and Windows.